.idea/compiler.xml

```diff
@@ -0,0 +1,13 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
can we add this folder to .gitignore if it isn't already? I think this is IDE specific stuff.

Reply:
removed the folder and added it to gitignore